account_payment_return_import_iso20022/wizard/camt_parser.py

```diff
@@ -123,6 +133,8 @@ def parse_payment_returns(self, ns, node):
                 payment_return['transactions'].extend(transactions)
                 subno = 0
                 for transaction in payment_return['transactions']:
+                    if not transaction.get('date'):
+                        transaction['date'] = entry_date
```

Comment:
This way of getting the date still looks confusing to me. Why not extracting the date in `parse_transactions` likt it is done for camt 054? Also, I see your example file has ns:RltdDts/ns:IntrBkSttlmDt and that seems to be the element with the date for camt 054? Any reason not to use that one for camt 053 too?

Reply:
I made a mistake in the example, in fact the 053 have no date element in the transaction, there is only in `Ntry` level

Reply:
I updated the test file